lib/checkPullRequestReview.js

```diff
@@ -287,6 +288,76 @@ const handlePullRequestReview = async (context) => {
   }
 };
 
+/**
+ * This function checks when PR author comments on the pull request.
+ * If pr author comments on the PR asking reviewers please to take a look (PTAL),
```

Comment:
please to take a look (PTAL) -> to please take a look (PTAL)

Reply:
Done.